device-types/Raritan/DSX2-16.yaml

```diff
@@ -0,0 +1,68 @@
+---
+manufacturer: Raritan
+model: RT-DSX2-16
+slug: rt-dsx2-16
+u_height: 1
+is_full_depth: false
+console-ports:
+  - name: ADMIN
+    type: usb-mini-a
+  - name: DVI
+    type: other
+  - name: Terminal
+    type: rj-45
+  - name: USB1
+    type: usb-a
+  - name: USB2
+    type: usb-a
+  - name: USB3
+    type: usb-a
```

Comment:
I am assuming these are used for Keyboards/Mice as this is a KVM.  This is not a valid use of the "console ports" for the device type library